packages/@react-aria/selection/src/useSelectableCollection.ts

```diff
@@ -218,7 +218,7 @@ export function useSelectableCollection(options: AriaSelectableCollectionOptions
       case 'a':
         if (isCtrlKeyPressed(e) && manager.selectionMode === 'multiple' && disallowSelectAll !== true) {
           e.preventDefault();
-          manager.selectAll();
+          manager.toggleSelectAll();
```

Comment:
After discussing with the team, I think we actually want `manager.selectAll()` instead, since it seems Ctrl+A should be 'select all': https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles/grid_role#keyboard_interactions

Sorry for the confusion! Feel free to revert this.

Reply:
No problem, I just think that `selectAll` makes more sense than `toggleSelectAll`.